sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js

```diff
@@ -6,15 +6,17 @@ const getUnlockedManager = require('./account-manager');
 async function run() {
     // Replace with an existing alias ID in your account
     const ALIAS_ID = '0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152'
-    
+
     try {
         const manager = await getUnlockedManager();
 
         const account = await manager.getAccount('0');
 
         await account.sync();
 
-        let tx = await account.destroyAlias(ALIAS_ID);
+        let tx = await account.burn({
+            aliases: [ALIAS_ID]
+        });
```

Comment:
Should the examples change if we keep the original functions?

Reply:
yea, to slowly phase them out i guess? IF we dont plan to phase them out i would say no

Reply:
I'm not sure we will

Reply:
Ok, changing them back